### PR TITLE
Add the reconciliation graph to HelmRelease detail page

### PR DESCRIPTION
**Pull request by jpellizzari**
Adds the graph to HelmReleases. While I was there, I standardized the automation detail pages into single components.

It might be easier to review this commit by commit to see the graph and standardization separately.

### CODE REVIEW

File: core/server/helm_release.go [hunk 1]
```diff
@@ -33,20 +33,20 @@ func (cs *coreServer) ListHelmReleases(ctx context.Context, msg *pb.ListHelmRele
 
 	var results []*pb.HelmRelease
 
-	for n, lists := range clist.Lists() {
+	for clusterName, lists := range clist.Lists() {
```
Comment (josecordaz) [inline core/server/helm_release.go:36]: 💯

File: ui/components/FilterableTable.tsx [hunk 2]
```diff
@@ -211,7 +211,9 @@ function FilterableTable({
     let prefix = "";
     if (query && !query.includes("filters") && url) prefix = "&?filters=";
     else if (url) prefix = "?filters=";
-    history.replace(location.pathname + prefix + encodeURIComponent(url));
+    history.replace(
+      (location.pathname || "") + prefix + encodeURIComponent(url)
```
Comment (joshri) [inline ui/components/FilterableTable.tsx:215]: 👍  - didn't realize there were cases when location.pathname wouldn't exist

File: ui/pages/v2/HelmReleasePage.tsx [hunk 3]
```diff
@@ -11,6 +13,31 @@ type Props = {
   className?: string;
 };
 
+function helmChartLink(helmRelease: HelmRelease) {
```
Comment (joshri) [inline ui/pages/v2/HelmReleasePage.tsx:16]: Does this need to be exported and then imported into HelmReleaseDetail now? Or deleted here? I'm just seeing the warning (defined but never used) and want to make sure this still works

File: ui/components/AutomationDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+export default styled(AutomationDetail).attrs({
+  className: AutomationDetail.name,
+})`
+  width: 100%;
+
+  ${Alert} {
+    margin-bottom: 16px;
+  }
+`;
```
Comment (joshri) [inline ui/components/AutomationDetail.tsx:117]: Love this thanks for unifying HelmRelease and Kustomization 🌞

**Review by joshri — APPROVED** [review]
🆒  COOL 🆒

File: ui/components/AutomationDetail.tsx (hunk 4 above)
Comment (jpellizzari) [reply ui/components/AutomationDetail.tsx:117]: Nice catch. I had turned off my `eslint` to save cycles. Not sure how that passed CI

File: ui/pages/v2/HelmReleasePage.tsx (hunk 3 above)
Comment (jpellizzari) [reply ui/pages/v2/HelmReleasePage.tsx]: Nice catch. I had turned off my `eslint` to save cycles. Not sure how that passed CI

**Pull request merged**